src/main/java/com/epam/wl/executor/Executor.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.wl.executor;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class Executor {
+    private final Connection connection;
+
+    public Executor(Connection connection) {
+        this.connection = connection;
+    }
+
+    public void execUpdate(String update) throws SQLException {
+        Statement stmt = connection.createStatement();
+        stmt.execute(update);
+        stmt.close();
+    }
+
+    public <T> T execQuery(String query, ResultHandler<T> handler) throws SQLException {
+        Statement stmt = connection.createStatement();
+        stmt.execute(query);
+        ResultSet result = stmt.getResultSet();
+        T value = handler.handle(result);
+        result.close();
+        stmt.close();
+
+        return value;
+    }
+
+}
```

Comment:
There's no reason to shorten the names, to make methods more readable I suggest renaming execUpdate and execQuery to executeUpdate and executeQuery